codecs/vp8_packet_test.go

```diff
@@ -0,0 +1,94 @@
+package codecs
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/pions/rtp"
+	"github.com/stretchr/testify/assert"
```

Comment:
Sorry mind dropping the dependency :( when users import it will get added to their modules (so trying to avoid adding things)